integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonDropTableCommand.scala

```diff
@@ -110,28 +97,6 @@ case class CarbonDropTableCommand(
 
       CarbonEnv.getInstance(sparkSession).carbonMetaStore.dropTable(identifier)(sparkSession)
 
-      if (carbonTable.hasDataMapSchema) {
-        // drop all child tables
-       val childSchemas = carbonTable.getTableInfo.getDataMapSchemaList
-
-        childDropCommands = childSchemas.asScala
-          .filter(_.getRelationIdentifier != null)
-          .map { childSchema =>
-            val childTable =
-              CarbonEnv.getCarbonTable(
-                TableIdentifier(childSchema.getRelationIdentifier.getTableName,
-                  Some(childSchema.getRelationIdentifier.getDatabaseName)))(sparkSession)
-            val dropCommand = CarbonDropTableCommand(
-              ifExistsSet = true,
-              Some(childSchema.getRelationIdentifier.getDatabaseName),
-              childSchema.getRelationIdentifier.getTableName,
-              dropChildTable = true
-            )
-            dropCommand.carbonTable = childTable
-            dropCommand
-          }
-        childDropCommands.foreach(_.processMetadata(sparkSession))
-      }
       val indexDatamapSchemas =
```

Comment:
can you please rename this to just `datamapSchemas` because it will have MV schemas also, so indexDataMapaSchemas will give wrong meaning

Reply:
fixed